DataFormats/CTPPSReco/src/classes_def.xml

```diff
@@ -39,4 +42,18 @@
   <class name="edm::Wrapper<edm::DetSetVector<TotemRPLocalTrack::FittedRecHit>>"/>
   <class name="std::vector<edm::DetSet<TotemRPLocalTrack::FittedRecHit> >"/>
   <class name="std::vector<TotemRPLocalTrack::FittedRecHit>"/>
+
+<!-- diamonds objects -->
+
+  <class name="CTPPSDiamondRecHit" ClassVersion="2">
+    <version ClassVersion="2" checksum="3356585814"/>
```

Comment:
please start from version=3

2 still works, but 3 is the recommended starting point